### PR TITLE
option stop dbus service

### PR DESCRIPTION
**Pull request by gutierri**

### CODE REVIEW

File: mpris2controller/controller.py [hunk 1]
```diff
@@ -54,8 +54,9 @@ def call(self, method_name):
 
 
 class Controller(dbus.service.Object):
-    def __init__(self, bus, call=None):
+    def __init__(self, bus, loop = None, call=None):
```
Comment (icasdri) [inline mpris2controller/controller.py]: Please follow PEP8 style for keyword arguments, i.e. `loop=None` :)

File: mpris2controller/controller.py [hunk 2]
```diff
@@ -171,6 +172,12 @@ def call_on_head_not_playing(self, method_name):
             else:
                 self.remove(cur)
 
+    @dbus.service.method(dbus_interface=MY_INTERFACE)
+    def StopLoop(self):
+        log.info("Stop main loop")
```
Comment (icasdri) [inline mpris2controller/controller.py]: Just one log message before (like the other methods) should be sufficient :)

Comment (icasdri) [reply mpris2controller/controller.py]: Also, maybe this method should be called `Quit` on DBUS ("stop loop" seems implementation-specific, and just "stop" might be confused with MPRIS stop)

**Review by icasdri — CHANGES_REQUESTED** [review]
Thanks for the pull request! Quick question: what is your use case for a Quit method?
Looking forward to pulling this :)

**Comment by gutierri** [comment]
In the meantime, I was testing things, developing and feeling a 
difficulty in stopping the service or reigning.
Putting a command to terminate the DBUS service seems to be an 
acceptable solution.

**Review by icasdri — APPROVED** [review]
Cool, thanks!

**Pull request merged**